src/components/cvaa-overlay/main-captions_window.js

```diff
@@ -79,7 +79,7 @@ class MainCaptionsWindow extends Component {
         <div className={style.title}>
           <Text id={'cvaa.title'} />
         </div>
-        <div>
+        <div role="group">
```

Comment:
did you try the menu element?

Reply:
yes, i mentioned in the description, it's not working on the nvda so i replaced to "group"

Reply:
i meant <fieldset> or optgrup

Reply:
so need to change the role from manuitem to "option"  or also change the element itself?